src/test/java/com/easypost/BatchTest.java

```diff
@@ -182,21 +182,21 @@ public void testCreateScanForm() throws EasyPostException, InterruptedException
     public void testAddRemoveShipment() throws EasyPostException {
         vcr.setUpTest("add_remove_shipment");
 
-        Shipment shipment = Shipment.create(Fixtures.oneCallBuyShipment());
+        Shipment shipment = vcr.client.shipment.create(Fixtures.oneCallBuyShipment());
 
-        Batch batch = Batch.create();
+        Batch batch = vcr.client.batch.create();
 
         List<Shipment> shipmentData = new ArrayList<>();
         Map<String, Object> params = new HashMap<>();
 
         shipmentData.add(shipment);
         params.put("shipments", shipmentData);
 
-        Batch batchWithAddedShipment = batch.addShipments(params);
+        Batch batchWithAddedShipment = vcr.client.batch.addShipments(params, batch.getId());
 
         assertEquals(1, batchWithAddedShipment.getNumShipments().intValue());
 
-        Batch batchWithoutShipment = batch.removeShipments(params);
+        Batch batchWithoutShipment = vcr.client.batch.removeShipments(params, batch.getId());
```

Comment:
Hmm, the ID of an object should always be the first parameter. Param ordering should be tied to the flow. We'll always need to know the object in question (Batch) and then we do stuff to it (add/remove shipments). Other params may come and go but if we are going this route, the ID will always be required.

This is true of all the previous instance based functions. You'll want to check them all.

Reply:
Not sure if I follow, but what is the benefit of swapping `param` and `id` as both parameters are required?

Reply:
Consistency across all functions like this. ID will always be required for these previous instance methods whereas some of these will have optional params. If ID comes first, then it's always the same order for all functions.

Reply:
Just looked at Go, indeed we the ID comes first before any other parameters